backend/package.json

```diff
@@ -4,14 +4,22 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node dist/index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node ./dist/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "ts-node-dev index.ts",
+    "tsc": "tsc"
   },
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "@types/cors": "^2.8.13",
     "@types/express": "^4.17.16",
     "@types/node": "^18.11.18",
+    "cors": "^2.8.5",
```

Comment:
i think cors actually is needed in regular dependencies, not just devDependencies.

Reply:
Ok, I moved it to dependencies.